asteroid/dsp/declipp.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def declipp(mix_np, est_np):
```

Comment:
I'm unsure about the name. Declipping is the task of restoring the signal after clipping. 
Maybe `normalize_estimates(estimates, mixture)` would make sense? The name is less pretty but more explicit.  

@jonashaag

Reply:
I will change the name but should I keep `est_np` as variable name to indicate that we only support `np.array` for now ?

Reply:
Yes, keep the `est_np` but if you can change the order of the variables it would be cool.